Sources/GoodExtensions/UIKit/UIViewExtensions.swift

```diff
@@ -316,3 +334,110 @@ public extension GRActive where Base: UIView {
 
 }
 
+// MARK: - Dimming
+
+public extension GRActive where Base: UIView {
+
+    private final class DimView: UIView {
+
+        init(intensity alpha: CGFloat = 0.3) {
+            super.init(frame: .zero)
+
+            translatesAutoresizingMaskIntoConstraints = false
+            backgroundColor = .black.withAlphaComponent(alpha)
+        }
+
+        required init?(coder: NSCoder) {
+            fatalError("init(coder:) has not been implemented")
+        }
+
+        func changeIntensity(_ alpha: CGFloat) {
+            backgroundColor = .black.withAlphaComponent(alpha)
+        }
+
+    }
+
+    func dim(intensity alpha: CGFloat = 0.3, duration: CGFloat = 0.3) {
+        undim()
+        let dimView = DimView(intensity: 0)
+
+        addSubview(dimView)
+        NSLayoutConstraint.activate([
+            dimView.topAnchor.constraint(equalTo: base.topAnchor),
+            dimView.leadingAnchor.constraint(equalTo: base.leadingAnchor),
+            dimView.trailingAnchor.constraint(equalTo: base.trailingAnchor),
+            dimView.bottomAnchor.constraint(equalTo: base.bottomAnchor)
+        ])
+
+        UIView.animate(withDuration: duration) {
+            dimView.changeIntensity(alpha)
+        }
+    }
+
+    func undim(duration: CGFloat = 0.3) {
```

Comment:
Neviem ci undim existuje. Asi dajme `removeDim` Znova v oboch pripadoch by som zrusil default animaciu.

Reply:
radsej by som potom premenoval oboje, nech to je rovnako `removeDim` a `removeBlur`